### PR TITLE
SEG-Y viewer

### LINKED ISSUE #123: SegyViewer

### PR DESCRIPTION
**Pull request by HansKallekleiv**
Resolves #123.

### CODE REVIEW

File: webviz_subsurface/containers/_segy_viewer.py [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+
+# pylint: disable=no-name-in-module
+from dash_colorscales import DashColorscales
+
+import webviz_core_components as wcc
+from webviz_config import WebvizContainerABC
+from webviz_config.webviz_store import webvizstore
+from webviz_config.common_cache import CACHE
+
+from ..datainput._seismic import load_cube_data, get_iline, get_xline, get_zslice
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/_segy_viewer.py]: Reorder import order according to https://www.python.org/dev/peps/pep-0008/#imports

File: webviz_subsurface/containers/_segy_viewer.py [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+
+
+class SegyViewer(WebvizContainerABC):
+    """### SegyViewer
+
+Inspired by SegyViewer (https://github.com/equinor/segyviewer) this container
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/_segy_viewer.py]: Change this to markdown link?

File: webviz_subsurface/containers/_segy_viewer.py [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+visualizes seismic 3D cubes with 3 plots (inline, xline and zslice).
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/_segy_viewer.py]: Write out in full? E.g. `xline` --> `crossline`.

File: webviz_subsurface/containers/_segy_viewer.py [hunk 4]
```diff
@@ -0,0 +1,522 @@
...
+
+        self.cube_id = "cube-id-{}".format(uuid4())
+        self.iline_map_id = "iline-map-id-{}".format(uuid4())
+        self.xline_map_id = "xline-map-id-{}".format(uuid4())
+        self.zline_map_id = "zline-map-id-{}".format(uuid4())
+        self.color_values_id = "color_values-id-{}".format(uuid4())
+        self.color_scale_id = f"{str(uuid4())}-color-scale-id"
+        self.color_range_btn = f"{str(uuid4())}-color-range-id"
+        self.zoom_btn = f"{str(uuid4())}-zoom-id"
+        self.state_store = "state-store-id-{}".format(uuid4())
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/_segy_viewer.py]: use f-strings

File: webviz_subsurface/containers/_segy_viewer.py [hunk 5]
```diff
@@ -0,0 +1,522 @@
...
+    def __init__(self, app, segyfiles: list, zunit="depth (m)", colors: list = None):
+
+        self.cube_id = "cube-id-{}".format(uuid4())
+        self.iline_map_id = "iline-map-id-{}".format(uuid4())
+        self.xline_map_id = "xline-map-id-{}".format(uuid4())
+        self.zline_map_id = "zline-map-id-{}".format(uuid4())
+        self.color_values_id = "color_values-id-{}".format(uuid4())
+        self.color_scale_id = f"{str(uuid4())}-color-scale-id"
+        self.color_range_btn = f"{str(uuid4())}-color-range-id"
+        self.zoom_btn = f"{str(uuid4())}-zoom-id"
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/_segy_viewer.py]: Do we need `str()`, or are f-strings converting to string for us?

File: webviz_subsurface/containers/_segy_viewer.py (hunk 1 above)
Comment (HansKallekleiv) [reply webviz_subsurface/containers/_segy_viewer.py]: Isnt it ordered?
standard => 3rdparty => local

**Pull request merged**